src/core/index.ts

```diff
@@ -82,4 +82,11 @@ function createRenderHook<
   return renderHook
 }
 
-export { createRenderHook, cleanup, addCleanup, removeCleanup, suppressErrorOutput }
+export {
+  createRenderHook,
+  cleanup,
+  addCleanup,
+  removeCleanup,
+  suppressErrorOutput,
+  setDefaultWaitOptions
```

Comment:
I feel like the worst kind of person to nitpick this on a review, but this one is in the opposite order to the others:

```suggestion
  setDefaultWaitOptions,
  suppressErrorOutput
```

I hate that this bothers me 😞